### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Formatting fixes and updated development section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,39 +18,30 @@ Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/
...
-
   3. Set the username that this integration is attached to.
   4. Generate a token used for an AWS SNS subscription. Copy this value as you will use it in a later step.
 
-2. Go to **System Console > Plugins > Management** and click **Enable** to enable the AWS SNS plugin.
+2. Go to **System Console > Plugins > Management** and select **Enable** to enable the AWS SNS plugin.
 
-Note: If you are running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+**Note:** If you're running Mattermost v5.11 or earlier, you must first go to the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-aws-SNS/releases), download the latest release, and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
```
Comment (justinegeffen) [inline README.md:33]: Is this still valid? We've removed similar content from other plugin docs such as the GitHub docs. Added to this, should we mention self-hosted vs. cloud and differences between them?

File: README.md [hunk 2]
```diff
@@ -18,39 +18,30 @@ Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/
```
Comment (justinegeffen) [inline README.md:18]: Propose adding a section of "Requirements" that include minimum server version (if applicable), license requirement etc.

**Comment by justinegeffen** [comment]
@mickmister are there any doc updates needed for the most recent release of this plugin?

**Comment by mickmister** [comment]
> are there any doc updates needed for the most recent release of this plugin?

The plugin now supports RDS events, which is worth noting https://github.com/mattermost/mattermost-plugin-aws-SNS/pull/49

File: README.md (hunk 2 above)
Comment (mickmister) [reply README.md:18]: Minimum server version is `5.2.0`. 0/5 on if we need to mention this. This plugin does not require a paid license to run.

File: README.md (hunk 1 above)
Comment (mickmister) [reply README.md:33]: I don't think there are any differences between cloud and on-prem for this plugin

**Review by mickmister — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (justinegeffen) [reply README.md:33]: Awesome. I'm happy to remove it pending @hanzei's feedback.

File: README.md (hunk 2 above)
Comment (justinegeffen) [reply README.md:18]: I've taken a look through the doc again and I don't think we need to call out when something doesn't require a paid license. I'm 0/5 on mentioning the minimum server version. @hanzei, can you be the tie-breaker?

File: README.md (hunk 1 above)
Comment (hanzei) [reply README.md:33]: I'm fine with removing this :+1:

File: README.md (hunk 2 above)
Comment (hanzei) [reply README.md:18]: I don't see a reason to call out requirements if there aren't any.

**Review by hanzei — APPROVED** [review]

**Pull request merged**